spec/models/contact_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Contact, type: :model do
+  describe 'initialize' do
```

Comment:
It's not your responsibility to test a method that was implemented by ActiveRecord